core/domain/role_services.py

```diff
@@ -131,3 +142,69 @@ def get_all_actions(role):
         role_actions.extend(get_all_actions(parent_role))
 
     return list(set(role_actions))
+
+
+def get_updatable_roles():
+    """Returns roles to which a user's role can be updated by admin interface.
+
+    Returns:
+        dict. A dict of all roles that a user can be given by admin.
+            key -> roles in format as stored in backed.
+            value -> roles in format to show users.
+    """
+    updatable_roles = {}
+    for role in PARENT_ROLES:
+        if role not in [feconf.ROLE_GUEST]:
+            updatable_roles[role] = get_human_readable_role(role)
+    return updatable_roles
+
+
+def get_viewable_roles():
+    """Returns roles by which users can be searched in admin interface.
```

Comment:
Ok. So, I should
- remove get_viewable_roles and get_updatable_roles from roles_services.
- Add VIEWABLE_ROLES and UPDATABLE_ROLES on top of roles_services.
- Directly call for get_human_readable_roles of these lists from admin.py.